CODE_OF_CONDUCT.md

```diff
@@ -10,30 +8,43 @@ We pledge to act and interact in ways that contribute to an open, welcoming, div
 
 These are the policies for upholding [our community’s standards of conduct](#our-standards). If you feel that a thread needs moderation, please contact the maintainers.
-
-1. Remarks that violate the community standards of conduct, including hateful, hurtful, oppressive, or exclusionary remarks, are not allowed. (Cursing is allowed, but never targeting another user, and never in a hateful manner).
-2. Remarks that maintainers find inappropriate, whether listed in the code of conduct or not, are also not allowed.
-3. Maintainers will first respond to such remarks with a warning.
-4. If the warning is unheeded, the user will be “kicked,” i.e., kicked out of the communication channel to cool off.
-5. If the user comes back and continues to make trouble, they will be banned, i.e., indefinitely excluded.
-6. Maintainers may choose at their discretion to un-ban the user if it was a first offense and they offer the offended party a genuine apology.
-7. If a maintainer bans someone and you think it was unjustified, please take it up with that maintainer, or with a different maintainer, in private. Complaints about bans in-channel are not allowed.
-8. Maintainers are held to a higher standard than other community members. If a maintainer creates an inappropriate situation, they should expect less leeway than others.
-
 The enforcement policies in the code of conduct apply to all official venues, including Discord channels, GitHub repositories, the Twitter community and all other forums.
+
+1. Hateful, hurtful, oppressive, or exclusionary remarks are not allowed.
+2. Personal attacks are out-of-line.
+   1. Disagree with specific actions and decisions in a constructive way.
+3. Avoid sweeping negative comments or jokes that could be construed as passive-aggressive complaints.
+   1. These are genuinely unhelpful and make everyone feel worse.
+   2. High level concerns are fine, but they should be actionable and specific.
+   3. Feedback is essential to help the project improve: please bring it up directly and try your best to accept decisions made.
+4. Nicknames or aliases that are offensive or overtly sexual are not allowed. You will be asked to change these.
+5. Spamming, low-grade trolling and flame-baiting make the community worse and are not allowed.
+   1. If it's disruptive, oddly uncomfortable or intended to confuse or prank other users, it's probably not welcome here.
+   2. Thoughtful political discussion is permitted in off-topic channels, but political memes are banned.
+6. Bevy is an all-ages community. This is not the place for explicitly sexual or gorey materials.
+
+Maintainers are held to a higher standard than other community members.
+If a maintainer violates community standards, they should expect less leeway than others.
+These policies apply to private interactions between community members as well as those in public: if someone has been harassing you in private, let us know.
+
+When the standards are violated, maintainers will:
```

Comment:
It's odd to see `If a maintainer violates community standards, they should expect less leeway than others.` immediately followed up with `When the standards are violated, maintainers will:`, which seems to suggest that even if maintainers are held to a higher standard, the only people that can hold them accountable is themselves.

While the venn diagram of maintainers and community moderators are currently a perfect circle, I think this should definitely be worded to say "moderators" over "maintainers".

Reply:
I'm not sure that's an improvement. Given that moderators may be the same as maintainers for a while, introducing the idea that they're seperate groups is unlikely to clarify matters. How about "uninvolved maintainers"?